src/StrategyConvexStaker.sol

```diff
@@ -0,0 +1,222 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.18;
+
+import {BaseStrategy, ERC20} from "@tokenized-strategy/BaseStrategy.sol";
+import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import {TradeFactorySwapper} from "@periphery/swappers/TradeFactorySwapper.sol";
+import {AuctionSwapper, Auction} from "@periphery/swappers/AuctionSwapper.sol";
+import {IConvexBooster, IConvexRewards} from "./interfaces/ConvexInterfaces.sol";
+
+contract StrategyConvexStaker is
```

Comment:
i saw this is a fork from:

https://github.com/dudesahn/curve-lp-v3/blob/dev/src/StrategyConvexStaker.sol

does he have a prod deployment of this or was it a WIP?